Gems/ROS2/gem.json

```
@@ -13,8 +17,12 @@
     "user_tags": [
         "ROS2"
     ],
+    "compatible_engines": [
+        “o3de-sdk==1.2.0”,
+        “o3de>=2.0.0”
```

This isn't correct double quotes.
This is using the smart quotes which isn't plain ascii
```suggestion
        "o3de-sdk==1.2.0",
        "o3de>=2.0.0"
```